File: 2-Ad-Hoc/ex2679.c
#include <stdio.h>

int main(){
    //variaveis
    int n;

    //entrada
    scanf("%d", &n);

    //processamento
    n += 1;
    if(n % 2 == 1){
        n += 1;
    }

    //saida
    printf("%d\n", n);

    return 0;
}

File: 2-Ad-Hoc/Ex2679.java
import java.util.Scanner;

public class Ex2679 {// para ser aceito no URI mude o nome da classe para Main
    public static void main(String args[]) {
        // variaveis
        Scanner scanner = new Scanner(System.in);
        int n;

        // entrada
        n = scanner.nextInt();

        // processamento
        n += 1;
        if (n % 2 == 1) {
            n += 1;
        }

        // saida
        System.out.println(n);
        scanner.close();
    }
}